### ISSUE TITLE
UWP release mode

### ISSUE THREAD

**Issue by senkia** [luberda-molinet/FFImageLoading#508]
Hi,

I'm having trouble applying any kind of transformations on both svg and normal images on UWP in release mode (or by checking the Compile with .NET Native tool chain in Debug mode). If I run the app with this settings I'm getting PlatformNotFoundExceptions and FileNotFoundExceptions, but they are handled and with this setting I don't have live diagnostics tool so I don't know where they are thrown. Consequently the images don't show up. Also the 

This only happens if I use any kind of transformations and only on UWP platform with the native tool chain. At first I suspected that it's a similar problem to this one:
https://github.com/luberda-molinet/FFImageLoading/issues/253

But my UWP project initializes Xamarin.Forms like this: 
```
var assembliesToInclude = new List<Assembly>()
{
  typeof(CachedImage).GetTypeInfo().Assembly,
  typeof(CachedImageRenderer).GetTypeInfo().Assembly,
  typeof(FFImageLoading.Transformations.BlurredTransformation).GetTypeInfo().Assembly,
};

CachedImageRenderer.Init();
Forms.Init(args, assembliesToInclude);
```

I've also tried to include one class from every FFImageLoading assembly in the assembly list but still no luck. The same happens if I run the samples from the GitHub repo. What do I miss?

**Comment by daniel-luberda** [comment]
> 

```C#
typeof(FFImageLoading.Forms.WinUWP.CachedImageRenderer).GetTypeInfo().Assembly
typeof(FFImageLoading.Forms.WinUWP.ImageSourceBinding).GetTypeInfo().Assembly
```

Related: 
- https://github.com/luberda-molinet/FFImageLoading/issues/332
- https://github.com/luberda-molinet/FFImageLoading/issues/301
- https://github.com/luberda-molinet/FFImageLoading/issues/298
- https://github.com/luberda-molinet/FFImageLoading/issues/253

**Comment by senkia** [comment]
I've tried adding the 
` typeof(FFImageLoading.Forms.WinUWP.ImageSourceBinding).GetTypeInfo().Assembly`
line but still no luck. I've checked the release output directory and I can see the proper FFImageLoading.Transformations dll targeting .NET Core.

**Comment by turibbio** [comment]
Any news? I'm having the same problem

**Comment by sacheinbaala** [comment]
I'm also facing the same problem. Do we have any updates on this?

**Comment by JKennedy24** [comment]
I am also seeing this, even when registering:

                    typeof(CachedImage).GetTypeInfo().Assembly,
                    typeof(CachedImageRenderer).GetTypeInfo().Assembly,
                    typeof(FFImageLoading.Transformations.CircleTransformation).GetTypeInfo().Assembly

**Comment by LocalJoost** [comment]
This still does not work. I have observed the following, using the full debug mode. When you run in debug, you see stuff like this happening:
Generating/retrieving image: 632C519A3C47BF4ABAD1FA2B40B752F6;40x0
Generating/retrieving image: 19C6881FE38603C4058AB1902BC52BC1;40x0
Generating/retrieving image: 3FAC25F9C04E2883367B0F47344E86D6;40x0
Wait for similar request for key: 85E08056EC3197CBE648EF63A7346B04;15x0
Wait for similar request for key: 85E08056EC3197CBE648EF63A7346B04;15x0
Generating/retrieving image: A3D13A0FFBD90AE5BF88D4C6D7BF7252;40x0
Generating/retrieving image: CB2FE0FE6E1E024C759595719FA858A1;40x0
Generating/retrieving image: 85E08056EC3197CBE648EF63A7346B04;15x0
Image loaded from cache: FDDF51A312384413A44B80F7A0A31B68;40x0
Generating/retrieving image: B487244112230F85DABF129904D744C7
Image loaded from cache: FDDF51A312384413A44B80F7A0A31B68;40x0
Generating/retrieving image: https://media.licdn.com/mpr/mprx/0_x2MbBAMdgjDLxcsRYm5LKrW2YaT2WRsAajk8_fNfuGSu4cBRoWkLCUWdD7DTwMNcVj5kQoHuY4SdVobcWbHH8oefd4SWVoQZobH5zAN7yGp8VgcAoi4kxXUnWD;90x0;CircleTransformation,borderSize=0,borderHexColor=
Generating/retrieving image: https://inspirepodcastsdev.blob.core.windows.net/topicassets/pro.png;60x0;CircleTransformation,borderSize=0,borderHexColor=

When you run for release, you see this
Generating/retrieving image: 632C519A3C47BF4ABAD1FA2B40B752F6
Generating/retrieving image: 19C6881FE38603C4058AB1902BC52BC1
Generating/retrieving image: 3FAC25F9C04E2883367B0F47344E86D6
Wait for similar request for key: 85E08056EC3197CBE648EF63A7346B04
Wait for similar request for key: 85E08056EC3197CBE648EF63A7346B04
Exception thrown: 'System.AggregateException' in System.Private.Threading.dll
Generating/retrieving image: A3D13A0FFBD90AE5BF88D4C6D7BF7252
Generating/retrieving image: CB2FE0FE6E1E024C759595719FA858A1
Generating/retrieving image: 85E08056EC3197CBE648EF63A7346B04
Image loaded from cache: FDDF51A312384413A44B80F7A0A31B68
Image loaded from cache: FDDF51A312384413A44B80F7A0A31B68
Generating/retrieving image: B487244112230F85DABF129904D744C7
Generating/retrieving image: https://media.licdn.com/mpr/mprx/0_x2MbBAMdgjDLxcsRYm5LKrW2YaT2WRsAajk8_fNfuGSu4cBRoWkLCUWdD7DTwMNcVj5kQoHuY4SdVobcWbHH8oefd4SWVoQZobH5zAN7yGp8VgcAoi4kxXUnWD;CircleTransformation,borderSize=0,borderHexColor=
Generating/retrieving image: https://inspirepodcastsdev.blob.core.windows.net/topicassets/cla.png;CircleTransformation,borderSize=0,borderHexColor=
Generating/retrieving image: https://inspirepodcastsdev.blob.core.windows.net/topicassets/mix.png;CircleTransformation,borderSize=0,borderHexColor=

See the difference? Then NNx0 (like ;40x0) is missing. What IS this?

**Comment by daniel-luberda** [comment]
Should be fixed after #651 merge.
